pkg/util/rowcodec/encoder.go

```diff
@@ -121,15 +127,19 @@ func (encoder *Encoder) reformatCols() (numCols, notNullIdx int) {
 		}
 	}
 	if r.large() {
-		largeNotNullSorter := (*largeNotNullSorter)(encoder)
-		sort.Sort(largeNotNullSorter)
+		if !isOrder {
```

Comment:
this changes inline the `Less` logic of those sorter interface into this method,  makes them coupled, if we later change `Less` logic of largeNotNullSorter, it's easy to forget to change it here

i guess you want to avoid one round of iteration when tempColIDs already sorted, if there are no significant performance gain after this change, it's more harmful to do it